deployer/src/handlers/mod.rs

```diff
@@ -459,9 +465,21 @@ pub async fn get_logs(
     Extension(deployment_manager): Extension<DeploymentManager>,
     Extension(claim): Extension<Claim>,
     CustomErrorPath((project_name, deployment_id)): CustomErrorPath<(String, Uuid)>,
+    Query(LogSize { mode, len }): Query<LogSize>,
 ) -> Result<Json<Vec<LogItem>>> {
+    let len = len.unwrap_or(0);
+    let mode: String = match mode {
+        Some(temp) => match temp {
+            LogMode::Head => "head".to_string(),
+            LogMode::Tail => "tail".to_string(),
+            _ => "all".to_string(),
+        },
+        None => "all".to_string(),
+    };
```

Comment:
We can derive [strum::Display](https://docs.rs/strum/latest/strum/derive.Display.html) for the LogMode enum, and we don't need to manually convert it to a string here. We can add a `#[strum(serialize_all = "lowercase")]` to make the string representation lower case.